Python/getargs.c

```diff
@@ -525,7 +542,7 @@ converttuple(PyObject *arg, const char **p_format, va_list *p_va, int flags,
                   arg == Py_None ? "None" : Py_TYPE(arg)->tp_name);
         return msgbuf;
     }
-    else {
+    else if (!PyTuple_Check(arg)) {
```

Comment:
It is better to check `PyTuple_Check()` first, as it is the most common use case.